### PR TITLE
Correct style to comply with cpplint

### PR DESCRIPTION
**Pull request by jacquelinekay**
This will also fix a few uncrustify and pyflakes errors that came up from the last large PR made to this repo.

### CODE REVIEW

File: include/rttest/utils.h [hunk 1]
```diff
@@ -64,25 +64,24 @@ static inline bool subtract_timespecs(const struct timespec * t1,
...
 }
 
-static inline unsigned long timespec_to_long(const struct timespec * t)
+static inline size_t timespec_to_long(const struct timespec * t)
 {
   return t->tv_sec * NSEC_PER_SEC + t->tv_nsec;
 }
 
-static inline void long_to_timespec(const unsigned long input, struct timespec * t)
+static inline void long_to_timespec(const size_t input, struct timespec * t)
```
Comment (dirk-thomas) [inline include/rttest/utils.h]: Not related to the PR but `const` doesn't make much sense on a primitive value passed by value.

Same below.

File: src/rttest.cpp [hunk 2]
```diff
@@ -303,7 +306,7 @@ int Rttest::read_args(int argc, char ** argv)
   }
 
   return this->init(iterations, update_period, sched_policy, sched_priority,
-    stack_size, filename);
+           stack_size, filename);
```
Comment (dirk-thomas) [inline src/rttest.cpp]: Please try wrapping after the open parenthesis instead.

File: src/rttest.cpp [hunk 3]
```diff
@@ -385,17 +387,17 @@ void Rttest::initialize_dynamic_memory()
   }
   this->sample_buffer.buffer_size = iterations;
   this->sample_buffer.latency_samples =
-    (int *) std::malloc(iterations * sizeof(int));
+    reinterpret_cast<int *>(std::malloc(iterations * sizeof(int)));
```
Comment (dirk-thomas) [inline src/rttest.cpp]: `void *` should be casted with `static_cast`.

Same below.

File: test/test_api.cpp [hunk 4]
```diff
@@ -12,12 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <gtest/gtest.h>
```
Comment (dirk-thomas) [inline test/test_api.cpp]: `gtest.h` is C++ and should therefore be included after C headers like `sys/resource.h`.

Comment (jacquelinekay) [inline test/test_api.cpp]: I will change it to its correct place, but ament_cpplint currently considers it a C header, which is why I reordered it in the first place.

Comment (dirk-thomas) [inline test/test_api.cpp]: Have you tried using quotes instead?

Comment (jacquelinekay) [inline test/test_api.cpp]: ah, that worked

**Comment by jacquelinekay** [comment]
Running Linux only because this library is disabled for OSX and Windows.
http://ci.ros2.org/job/ros2_batch_ci_linux/486/

**Comment by jacquelinekay** [comment]
The error message for this test failure, although I could be wrong. @dirk-thomas

http://ci.ros2.org/job/ros2_batch_ci_linux/486/testReport/tests/test_code_format/test_flake8/

**Comment by dirk-thomas** [comment]
It looks like `flake8` has updated its API and `osrf_pycommon` needs to be fixed.

**Comment by jacquelinekay** [comment]
Rerunning CI:
http://ci.ros2.org/job/ros2_batch_ci_linux/514/

**Comment by jacquelinekay** [comment]
this PR now needs ros2/demos#37 to build. (this change is also included in the allocator_template branch of this repository.)

File: src/rttest.cpp [hunk 5]
```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <rttest.h>
```
Comment (dirk-thomas) [inline src/rttest.cpp]: This as well as `utils.h` should not be importable without a namespace / folder in the first place.

https://github.com/ros2/rttest/blob/2db2779fbdff6934797d645012ff2bb302e99dd5/CMakeLists.txt#L18 should not point to `include/rttest` but to `include`.

**Comment by jacquelinekay** [comment]
http://ci.ros2.org/job/ros2_batch_ci_linux/520/

The only test failures are now due to missing dependencies, which will be fixed by other PRs.

**Pull request merged**